vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2392,6 +2393,28 @@ static bool ignore_ = false;
     return "";
   }
 
+  std::vector<const std::string> GetYoutubeSpecificPaths() {
```

Comment:
I think this one should be in media file as well as it will not be used anywhere else. Also can you define this just as variable, I think we don't need function.

Reply:
Done